frontend/src/search/Searchbar.tsx

```diff
@@ -0,0 +1,119 @@
+import React from 'react';
+import TextField from '@material-ui/core/TextField';
+import Autocomplete from '@material-ui/lab/Autocomplete';
+import Grid from '@material-ui/core/Grid';
+import Typography from '@material-ui/core/Typography';
+import { makeStyles } from '@material-ui/core/styles';
+import { SearchResultOption } from './SearchResultOption';
+
+const useStyles = makeStyles((theme) => ({
+  icon: {
+    color: theme.palette.text.secondary,
+    marginRight: theme.spacing(2),
+  },
+  searchbar: {
+    backgroundColor: 'white',
+    borderRadius: 4,
+  },
+  searchbarWrapper: {
+    alignSelf: 'flex-end',
+    marginLeft: 'auto',
+    marginTop: 10,
+    marginBottom: 10,
+  },
+}));
+
+export default function Searchbar(): JSX.Element {
+  const classes = useStyles();
+  const [value, setValue] = React.useState<SearchResultOption | null>(null);
+  // TODO use the inputValue for the service call
+  const [inputValue, setInputValue] = React.useState(''); // eslint-disable-line @typescript-eslint/no-unused-vars
+  const [options, setOptions] = React.useState<SearchResultOption[]>([]);
+
+  /* const searchService = useService(SearchService, null);
+
+  const executeSearch = React.useMemo(
+    () => (input: string, searchService: SearchService) => {
+      // Service call here
+    },
+    []
+  );
+
+  React.useEffect(() => {
+    let active = true;
+
+    if (inputValue === '') {
+      setOptions(value ? [value] : []);
+      return undefined;
+    }
+
+    executeSearch(inputValue)
+      .then((response) => response.blob())
+      .then((result) => {
+        if (active) {
+          console.log(result);
+          const resultOptions = [] as SearchResultOption[]; // convertSearchResultToOptions(result as SearchResult);
+          let newOptions = [] as SearchResultOption[];
+
+          if (value) {
+            newOptions = [value];
+          }
+
+          if (resultOptions) {
+            newOptions = [...newOptions, ...resultOptions];
+          }
+
+          setOptions(newOptions);
```

Comment:
I think this won't be executed in the `useEffect` runcycle since it's an async operation